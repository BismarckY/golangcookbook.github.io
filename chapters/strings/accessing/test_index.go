package main

import "fmt"

func main() {
	s := "Hello World"
	fmt.Println(s[:5])           // the first 5 characters
	fmt.Println(s[:len(s)-6])    // the same, but counting back from the end
	fmt.Println(s[1:])           // all except the first character
	fmt.Println(s[len(s)-5:])    // the last 5 characters
	fmt.Println(s[1 : len(s)-1]) // all except the first and last
}
