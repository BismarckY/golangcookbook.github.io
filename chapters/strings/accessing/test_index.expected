Hello
Hello
ello World
World
ello Worl
